sklearn/calibration.py

```diff
@@ -1323,6 +1329,9 @@ def from_predictions(
         Read more about calibration in the :ref:`User Guide <calibration>` and
         more about the scikit-learn visualization API in :ref:`visualizations`.
 
+        For an example of usage, see:
+        :ref:`sphx_glr_auto_examples_calibration_plot_compare_calibration.py`.
```

Comment:
I'd put this in the "Examples" section further down in the docstring. That is what has been done for the other examples, consistency for the win :D

Reply:
Thanks for the example. DONE.